compiler/native/compile_test.go

```diff
@@ -538,6 +539,9 @@ func TestNative_Compile_StepsPipeline(t *testing.T) {
 	}
 
 	if !reflect.DeepEqual(got, want) {
+		if diff := cmp.Diff(want, got); diff != "" {
+			t.Errorf("MakeGatewayInfo() mismatch (-want +got):\n%s", diff)
+		}
```

Comment:
Is this supposed to be here?

If so, then maybe we should update the error message?

```suggestion
		if diff := cmp.Diff(want, got); diff != "" {
			t.Errorf("Compile() mismatch (-want +got):\n%s", diff)
		}
```